conf/alluxio-site.properties.template

```diff
@@ -75,6 +75,7 @@ alluxio.user.failed.space.request.limits=3
 alluxio.user.file.buffer.bytes=1MB
 alluxio.user.file.master.client.threads=10
 alluxio.user.file.readtype.default=CACHE_PROMOTE
+alluxio.user.file.cache.incomplete.block=false
```

Comment:
Not sure if this might be confusing, since we have a concept of `complete` and `incomplete` files.